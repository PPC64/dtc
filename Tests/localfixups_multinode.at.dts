/dts-v1/;
/plugin/;

/{
   quxnode {
      qux = <&BAZ>;
   };
   quuxnode {
      quux = <1 2 3 4 &FOO>;
   };
   FOO: foo { };
   BAZ: baz { };
};
