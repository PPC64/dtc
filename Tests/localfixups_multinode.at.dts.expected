/dts-v1/;

/  {

	quxnode {

		qux = <0x2>;
	};
	quuxnode {

		quux = <0x1 0x2 0x3 0x4 0x1>;
	};
	foo {

		phandle = <0x1>;
	};
	baz {

		phandle = <0x2>;
	};
	__symbols__ {

		BAZ = "/baz";
		FOO = "/foo";
	};
	__local_fixups__ {

		quxnode {

			qux = <0x0>;
		};
		quuxnode {

			quux = <0x10>;
		};
	};
};
